src/workers/notifications/postBookmarkReminder.ts

```diff
@@ -1,18 +1,11 @@
-import { messageToJson } from '../worker';
 import { Bookmark } from '../../entity';
 import { NotificationType } from '../../notifications/common';
-import { NotificationWorker } from './worker';
+import { generateTypedNotificationWorker } from './worker';
 import { buildPostContext } from './utils';
 
-interface Data {
-  postId: string;
-  userId: string;
-}
-
-const worker: NotificationWorker = {
-  subscription: 'api.post-bookmark-reminder-notification',
-  handler: async (message, con) => {
-    const { postId, userId }: Data = messageToJson(message);
+const worker = generateTypedNotificationWorker({
```

Comment:
Isn't it missing the type parsing here somewhere?
EG:
```
const worker: TypedWorker<'comment-upvoted'> = {
```

Reply:
We don't have to anymore. The types will automatically be inferred through the `subscription` property.

Reply:
Ah perfect 👌